recipes/obsplus/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "obsplus" %}
+{% set version = "0.0.6" %}
+{% set sha256 = "40374fe113e03ed41b807395fda6b5a513d9f4578da384d2de08cc0ec11fd4b5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pip
+  host:
+    - python >=3.6
+    - numpy >=1.13.0
+    - obspy >=1.1.0
+    - pandas >=0.24.0
+    - progressbar2
+    - scipy
+    - setuptools
+    - pytables
```

Comment:
Do you actually need most of these just to install?

Reply:
Honestly I don't know. I am still not sure what the difference between `build` and `host` is even after reading the docs several times. I will try removing some of them.

Reply:
Yeah, it can be a little confusing, especially if you are not used to cross-compiling.  Build is for build tools (like compilers) that need to run natively on the platform that is being used to build (e.g. linux64). Host is for libraries that will be linked against on the target platform that the code will be installed onto (eg. linux-aarch64 or osx64). Both build and host are needed when you install a package, but they have important differtent meaning for what the intent of the produced shared object libraries is. 

For noarch packages, build isn't important, and so host list just the deps that are needed for installing the package

Reply:
Yeah, it can be a little confusing, especially if you are not used to cross-compiling.  Build is for build tools (like compilers) that need to run natively on the platform that is being used to build (e.g. linux64). Host is for libraries that will be linked against on the target platform that the code will be installed onto (eg. linux-aarch64 or osx64). Both build and host are needed when you install a package, but they have important differtent meaning for what the intent of the produced shared object libraries is. 

For noarch packages, build isn't important, and so host list just the deps that are needed for installing the package

Reply:
Ok so the python (with version restrictions) and pip should be all that is required in this section no?

Reply:
Yes, only python and pip are needed.  This only needs to build on linux because it is noarch: python